config.js

```diff
@@ -12,8 +14,11 @@ module.exports = {
 	{ id: 6, name: 'Pizza Pazza', url: 'http://pizzapazza.sk/?page_id=645', module: 'pizzapazza', post: 'prevadzka2=1' },
 	{ id: 7, name: 'Kamenica - Corleone Pizza', url: 'http://www.pizzacorleone.sk/obedove-menu.html', module: 'kamenica' },
 	{ id: 8, name: 'Numero Uno', url: 'http://www.numerouno.sk/denne-menu/', module: 'numerouno' },
-	{ id: 9, name: 'Engerau restaurant', url: 'https://www.zomato.com/sk/bratislava/engerau-restaurant-petr%C5%BEalka-bratislava-v/denn%C3%A9-menu', module: 'engerau' },
-	{ id: 10, name: 'Lokálka', url: 'http://www.lokalka.sk/kopcianska-2/', module: 'lokalka' }
+	{ id: 10, name: 'Lokálka', url: 'http://www.lokalka.sk/kopcianska-2/', module: 'lokalka' },
+	// jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+	{ id: 9, name: 'Engerau restaurant', url: 'https://developers.zomato.com/api/v2.1/dailymenu?res_id=16507706', webUrl: 'https://www.zomato.com/bratislava/engerau-restaurant-petr%C5%BEalka-bratislava-v/daily-menu', module: 'engerau', headers: { 'user_key' : userKey }, noCache: true },
```

Comment:
the zomato agreement policy contains a statement which forbids to store, cache etc .. any data acquired with their services (probably so you cant copy the database or abuse the data). its not stated what they mean by storing the data or if there is a specific amount of it that you can store. personally i think the policy is badly written. its up to you if you want to turn on the caching or not.